storage/replica_command.go

```diff
@@ -1045,10 +1044,10 @@ func (r *Replica) LeaderLease(batch engine.Engine, ms *engine.MVCCStats, args pr
 		log.Infof("range %d: new leader lease %s", rangeID, args.Lease)
 	}
 
-	// Gossip configs in the event this range contains config info.
-	r.maybeGossipConfigsLocked(func(configPrefix proto.Key) bool {
-		return r.ContainsKey(configPrefix)
-	})
+	// Gossip system config if this range is in the system span.
```

Comment:
The system span is always contained within a single range, correct? This comment is confusing as written. Perhaps something like: `The system span is always contained entirely within a single range. If this is that range gossip the system config.`